website/docs/reference/warehouse-profiles/singlestore-profile.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "SingleStore Profile"
+---
+
+:::info Vendor-supported plugin
+
+Certain core functionality may vary. If you would like to report a bug, request a feature, or contribute, you can check out the linked repository and open an issue.
+
+:::
+
+## Overview of dbt-singlestore
+
+**Maintained by:** SingleStore, Inc.  
+**Source:** [Github](https://github.com/memsql/dbt-singlestore)  
+**Core version:** v1.0.0 and newer  
+**dbt Cloud:** Not Supported  
+**dbt Slack channel:** [Slack](https://getdbt.slack.com/archives/C02V2QHFF7U)  
+**SingleStore version:** v7.5 and newer  
+
+
+## Installation and Distribution
+
+SingleStore dbt adapter is managed in its own repository, [dbt-singlestore](https://github.com/memsql/dbt-singlestore). You can use `pip` to install the SingleStore adapter:
+
+```
+pip install dbt-singlestore
+```
+
+Alternatively, you can clone the repository contents to a folder, and run `pip install .` from it. For other information including support of dbt features by SingleStore, see the [GitHub README](https://github.com/memsql/dbt-singlestore#readme).
```

Comment:
If there indeed is other configuration, beyond profile set up, we prefer that it lives on this docs site instead of the dbt-singlestore's README

```suggestion
Alternatively, you can install the package from GitHub with:
    ```zsh
    pip install git+https://github.com/memsql/dbt-singlestore.git
    ```
For other information including support of dbt features by SingleStore, see the [GitHub README](https://github.com/memsql/dbt-singlestore#readme).
```